### ISSUE TITLE
Component: Dropdown Required ARIA attribute not present: aria-expanded

### ISSUE THREAD

**Issue by Draganlazarev90** [primefaces/primeng#14967]
### Describe the bug

aria-expanded property is missing from span role="combobox"

### Environment

Primeng 17

### Angular version

Primeng 17

### PrimeNG version

Primeng 17.8

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0
